pkgs/applications/file-managers/tuifimanager/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, python3Packages, fetchFromGitHub }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "tuifimanager";
+  version = "2.1.3";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "GiorgosXou";
+    rev = "1470b7b7d95fbf05548bbd60c8dacfc7fe4e9407";
```

Comment:
Git repo doesn't have tags so I need to specify a commit. This commit refers to the commit "v.2.1.3"